### DISCUSSION TITLE
Approved but not recieved the benefits

### DISCUSSION THREAD

**Discussion by SakshamShandilya** [community/community#135085] [GitHub Education]
### Hi everyone,

It's been more than 3 years but I have not received my benefits please help @

**Comment by Nilstam428** [comment]
Try to login with your college domain mail id which you have shared to github while activating your student benefit.

**Reply by SakshamShandilya** [reply]
<img width="478" alt="Screenshot 2024-08-08 at 5 14 22 PM" src="https://github.com/user-attachments/assets/4498f404-111f-4937-ba16-0b8f022aaea6">

Doing it with that

**Comment by Nilstam428** [comment]
But you're saying more than 3 years?
